pkg/util/util_test.go

```diff
@@ -30,13 +30,13 @@ import (
 
 var _ = Describe("GetMasterURL", func() {
 	BeforeEach(func() {
-		os.Setenv(common.EnvKubernetesServiceHost, "127.0.0.1")
-		os.Setenv(common.EnvKubernetesServicePort, "443")
+		os.Setenv(common.EnvKubernetesServiceHost, "127.0.0.1") //nolint:errcheck
```

Comment:
```suggestion
		Expect(os.Setenv(common.EnvKubernetesServiceHost, "127.0.0.1")).NotTo(HaveOccurred())
```

We can use `Expect(err).NotTo(HaveOccurred())` to avoid skipping the lint check.

Reply:
Done.